stream/src/main/java/org/apache/kafka/clients/processor/Chooser.java

```diff
@@ -15,9 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.kafka.stream;
-
-import org.apache.kafka.stream.internals.RecordQueue;
+package org.apache.kafka.clients.processor;
 
 public interface Chooser {
 
```

Comment:
Records are prioritized by the whole buffer contents (queues) not by individual record timestamp. We can move many public things to private or internals if we decide not to expose them. We just need to decide.